Formula/ponyc.rb

```diff
@@ -1,10 +1,9 @@
 class Ponyc < Formula
   desc "Object-oriented, actor-model, capabilities-secure programming language"
   homepage "http://www.ponylang.org"
-  url "https://github.com/ponylang/ponyc/archive/0.2.1.tar.gz"
-  sha256 "cb8d6830565ab6b47ecef07dc1243029cef962df7ff926140022abb69d1e554e"
-  revision 1
-  head "https://github.com/ponylang/ponyc.git"
+  url "https://github.com/ponylang/ponyc.git",
+    :revision => "aafebac938273d4786c02dfb2ba9ec7a164675e7"
```

Comment:
This might be a problem for us, as we don't normally package a somewhat arbitrarily chosen commit and advertise it as the stable release shipped by Homebrew. Best solution would be to contact Pony upstream and find out about their release plans and LLVM support.

If we can't have a stable release that works with LLVM 3.8 or some other release that is blessed by upstream for use with LLVM 3.8, we might have to temporarily boneyard this formula until upstream has official support for LLVM 3.8. (Or we go to the trouble of vendoring LLVM 3.6.)